client/main.js

```diff
@@ -0,0 +1,27 @@
+import { Meteor } from 'meteor/meteor';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render } from 'react-dom';
+
+import App from './../imports/ui/containers/App';
+import styles from './main.css';
+
+Meteor.startup(() => {
+  Meteor.autorun(() => {
+    ReactDOM.render(<App />, document.getElementById('app'));
```

Comment:
``` ReactDOM.render(<App />, document.getElementById('app'));``` 
Does not need to be wrapped in:
```Meteor.autorun(() => ```

From the docs: 
https://docs.meteor.com/api/tracker.html
"Tracker.autorun allows you to run a function that depends on reactive data sources, in such a way that if there are changes to the data later, the function will be rerun."

In Meteor, these are reactive data-sources:
- Collection queries
- Any code that uses Meteor's Session